### DISCUSSION TITLE
Problema com o código.

### DISCUSSION THREAD

**Discussion by DeymonSR** [community/community#75891] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Estou com problema para compreender essa exercício alguém pode me ajudar?

Utilize o operador lógico correto: feliz ou triste? 🎭
Abaixo existem duas variáveis, uma se chama local e a outra estadoEmocional:

let local = 'praia';
let estadoEmocional;

A variável local representa o lugar em que a pessoa está no momento;
A variável estadoEmocional representa qual o estado emocional da pessoa ao estar nesse local;
Crie uma lógica para saber o valor de estadoEmocional:

O valor da variável estadoEmocional deve ser uma verificação se o valor da variável local for igual a praia OU a variável local seja igual a cinema;

**Comment by Akash1134** [comment]
Hi @DeymonSR , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
